src/server/sockets.ts

```diff
@@ -1,11 +1,32 @@
 import { io } from './configs/socket.config.js';
+import { generateTournamentPin } from './helpers/socket.helper.js';
+
+const tournamentPool: any = {
+  // tPin: [
+  //     socketIds,
+  //     socketIds,
+  // ],
+    // tPin: [
+  //     socketIds,
+  //     socketIds,
+  // ]
+}
 
 export default io.on("connection", (socket) => {
     console.log("user connected, ", socket.id)
 
-    socket.on('create-room', (room) => {
-      socket.join(room)
-      console.log('id: ', room)
+    socket.on('create-tournament', () => {
```

Comment:
try to model this on after my create match ... with the obvious differences of you are dealing with a tournament im dealing with a match

Reply:
will do this on my next branch